ddtrace/span.py

```diff
@@ -37,6 +38,9 @@
     from .tracer import Tracer
 
 
+MetaKey = Union[Text, bytes]
+MetaDict = Dict[MetaKey, Text]
```

Comment:
```suggestion
MetaKeyType = Union[Text, bytes]
MetaDictType = Dict[MetaKey, Text]
```
nit, usually easier to recognize those with `Type` at the end. Not a big deal though.

Also, be careful that this might become public API very soon if anyone starts using type hints with our code base.

Reply:
Agreed on both points. Was thinking of making these internal. Went ahead with that. We can drop the `Type` suffix then if we decide at some point it's not helpful.